pkgs/os-specific/darwin/apple-sdk/print-reexports/main.c

```diff
@@ -0,0 +1,129 @@
+#include <stdio.h>
```

Comment:
Can we add a small synposis as a comment what `print-reexports` is supposed to do? And why it's needed?

Reply:
Added a comment.

More generally, I don't know if this approach to cross-linking is desirable or feasible long term. I'll likely revisit it when working on arm64 support.